src/redturtle/prenotazioni/upgrades.py

```diff
@@ -184,3 +187,8 @@ def to_1401(context):
             for workflow_action in workflow_action_conditions:
                 if workflow_action.transition == "publish":
                     workflow_action.transition = "confirm"
+
+
+def to_1402(context):
+    # load new content rules
+    context.runImportStepFromProfile(CONTENT_RULES_EVOLUTION_PROFILE, "contentrules")
```

Comment:
Come avevamo discusso, facendo così rischi di andare a sovrascrivere una rule già esistente con lo stesso id.

Forse nell'upgrade-step conviene crearla a mano e/o controllare che non ci sia già una regola con quell'id.

Esempio di codice: https://github.com/PloneGov-IT/rer.groupware.notify/blob/master/rer/groupware/notify/events.py

@mamico pareri?

Reply:
Abbiamo deciso di lasciare questo profilo visto che le regole create dal plone hanno un id specifico es: "rule-1"